mpqp/execution/providers/ibm.py

```diff
@@ -21,7 +21,6 @@
 from mpqp.execution.devices import AZUREDevice, IBMDevice
 from mpqp.execution.job import Job, JobStatus, JobType
 from mpqp.execution.result import Result, Sample, StateVector
-from mpqp.execution.simulated_devices import IBMSimulatedDevice
```

Comment:
Question: Why did we remove the import of this mpqp class to deal with it as it was a class from an external package? Did cause any circular import problem? Or maybe for performance issues ? (the generation of fake devices?)

Reply:
It's a performance issue: in order to generate the enum values, you have to import qiskit, and we don't want that to happen on default import (import mpqp)

Reply:
Very good. I resolve